security-release-process.md

```diff
@@ -44,30 +45,34 @@ The Fix Lead will give the Fix Team access to a private security repository to d
-The Release Managers will create a new patch release branch from the latest patch release tag + the fix from the security branch. As a practical example if v1.5.3 is the latest patch release in gardener.git a new branch will be created called v1.5.4 which includes only patches required to fix the issue.
+The Release Managers will create a new patch release branch from the latest patch release tag + the fix from the security branch. As a practical example if v0.12.0 is the latest patch release in gardener.git a new branch will be created called v0.12.1 which includes only patches required to fix the issue.
 The Fix Lead will cherry-pick the patches onto the master branch and all relevant release branches. The Fix Team will [LGTM](https://github.com/lgtmco/lgtm) and merge.
 The Release Managers will merge these PRs as quickly as possible. Changes shouldn't be made to the commits even for a typo in the CHANGELOG as this will change the git sha of the already built and commits leading to confusion and potentially conflicts as the fix is cherry-picked around branches.
-The Fix Lead will request a CVE from SAP Global Security via email to [cna@sap.com](mailto:cna@sap.com) with all the relevant information (description, potential impact, affected version, fixed version, CVSS v3 base score and supporting documentation for the CVSS score) for every vulnerability.
-The Fix Lead will e-mail the Gardener mailing list [gardener@googlegroups.com](mailto:gardener@googlegroups.com) now that everything is public announcing the new releases, the CVE number, the location of the binaries, and the relevant merged PRs to get wide distribution and user action. As much as possible this e-mail should be actionable and include links how to apply the fix to users environments; this can include links to external distributor documentation. The recommended target time is 4pm UTC on a non-Friday weekday. This means the announcement will be seen morning Pacific, early evening Europe, and late evening Asia.
+The Fix Lead will request a CVE from the SAP Product Security Response Team via email to [cna@sap.com](mailto:cna@sap.com) with all the relevant information (description, potential impact, affected version, fixed version, CVSS v3 base score and supporting documentation for the CVSS score) for every vulnerability.
+The Fix Lead will inform the [Gardener mailing list](#communication-channel) that everything is public announcing the new releases, the CVE number (if available), the location of the binaries, and the relevant merged PRs to get wide distribution and user action. As much as possible this e-mail should be actionable and include links how to apply the fix to users environments; this can include links to external distributor documentation. The recommended target time is 4pm UTC on a non-Friday weekday. This means the announcement will be seen morning Pacific, early evening Europe, and late evening Asia.

```

Comment:
this sentence seems broken

s/The Fix Lead will inform the [Gardener mailing list](#communication-channel) that everything is public announcing the new releases, the CVE number (if available), the location of the binaries, and the relevant merged PRs to get wide distribution and user action./The Fix Lead will inform the [Gardener mailing list](#communication-channel) and announce the new releases, the CVE number (if available), the location of the binaries, and the relevant merged PRs to get wide distribution and user action./